e2e-tests/gef/cypress/integration/unissued-facility-change-from-preview-MIN.spec.js

```diff
@@ -0,0 +1,254 @@
+import relative from './relativeURL';
+
+const CONSTANTS = require('../fixtures/constants');
+
+const { add, format } = require('date-fns');
```

Comment:
I've seen this being added it with require. I believe we can change it to `import` instead? also, this line needs to be first as it's importing an external package. this occurs in other files as well from this PR

Reply:
done